readme.md

```diff
@@ -106,6 +106,18 @@ const tasks = new Listr([
 	{
 		title: 'Failure',
 		task: () => Promise.reject(new Error('Bar'))
+	},
+	{
+		title: 'Output',
```

Comment:
Move this example to [this](https://github.com/SamVerschueren/listr#task-object) section as well.